app/src/test/java/com/concordium/wallet/data/util/ParamsDeserializerShould.kt

```diff
@@ -0,0 +1,106 @@
+package com.concordium.wallet.data.util
+
+import com.concordium.wallet.data.walletconnect.Params
+import com.concordium.wallet.data.walletconnect.ParamsDeserializer
+import com.google.gson.GsonBuilder
+import com.google.gson.JsonSyntaxException
+import org.junit.Assert
+import org.junit.Test
+
+/**
+ * Unit Test for the [ParamsDeserializer]
+ */
+class ParamsDeserializerShould {
```

Comment:
Still missing a test, where the output returned from deserialize method, is verified to be correct.

Reply:
In example, in the arrange part of your test, define a variable named "paramWithSchemaAsString". Then make another variable named "expectedOutput". 

Then later in your test - after you deserialize "paramWithSchemaAsString", assert that paramsFromStringSchema?.schema?.value is equal to the expectedOutput. 

This should be done for both schemaAsString and schemaAsObject in two separate tests.